xtable-core/src/main/java/org/apache/xtable/delta/DeltaConversionTarget.java

```diff
@@ -66,9 +66,9 @@
 import org.apache.xtable.spi.sync.ConversionTarget;
 
 public class DeltaConversionTarget implements ConversionTarget {
-  private static final String MIN_READER_VERSION = String.valueOf(1);
+  private static final String MIN_READER_VERSION = String.valueOf(3);
```

Comment:
Handle this gracefully by calling the function which upgrades version in delta codebase, this was the pending comment from the previous PR that wasn't addressed. 

https://github.com/apache/incubator-xtable/pull/428